core/src/eval/tests.rs

```diff
@@ -338,6 +338,13 @@ fn substitution() {
         .unwrap();
     assert_eq!(
         subst(&eval_cache, t, &initial_env, &env),
-        parse("match {'x => [1, 1], 'y => (if false then 1 else \"Glob2\"), 'z => {id = true, other = false}} true").unwrap()
+        parse(
+            "match {\
+                'x => [1, 1], \
+                'y => (if false then 1 else \"Glob2\"), \
+                'z => {id = true, other = false}\
+            } true"
+        )
+        .unwrap()
```

Comment:
1. I didn't to avoid adding extra whitespace to the test. Since I was making a lot of changes I tried to avoid changing the meaning of any code. In this case, yes. We can. It would technically add some whitespace to the string, but eh ¯\\\_(ツ)\_/¯.
2. I have vertical ruler at col. 100 on vscode, and I'm guessing the other one wasn't past the 100 mark. Again, rapid fire changes.